backend/de.metas.shipper.gateway.spi/src/main/java/de/metas/shipper/gateway/spi/model/PackageDimensions.java

```diff
@@ -45,4 +47,9 @@ private PackageDimensions(final int lengthInCM, final int widthInCM, final int h
 		this.heightInCM = heightInCM;
 	}
 
+	public static PackageDimensions unspecified()
+	{
+		return new PackageDimensions(UNSPECIFIED_DIMENSION, UNSPECIFIED_DIMENSION, UNSPECIFIED_DIMENSION);
```

Comment:
Why create an instance each time instead of returning a constant?

Reply:
changed to use constant